camayoc/tests/qcs/cli/test_scans.py

```diff
@@ -113,270 +106,165 @@ def setup_sources():
-    })
-    match = re.match(r'Scan "(\d+)" started.', result)
-    assert match is not None
-    scan_job_id = match.group(1)
-    wait_for_scan(scan_job_id, timeout=1200)
-    result = scan_job({
-        'id': scan_job_id,
-    })
-    assert result['status'] == 'completed'
-    report_id = result['report_id']
-    assert report_id is not None
-    output_file = 'out.json'
-    report = report_detail({
-        'json': None,
-        'output-file': output_file,
-        'report': report_id,
-    })
-    with open(output_file) as report_data:
-        report = json.load(report_data)
-        assert report.get('sources', []) != []
+
+    source_output = source_show_output({'name': source_name})
+    source_output = json.loads(source_output)
+
+    scan_show_result = scan_show({'name': scan_name})
+    scan_show_result = json.loads(scan_show_result)
+    expected_result = {'id': scan_show_result['id'],
+                       'name': scan_name,
+                       'options': {
+                           'disabled_optional_products': {
+                               'jboss_brms': False,
+                               'jboss_eap': True,
+                               'jboss_fuse': False},
+                           'enabled_extended_product_search': {
+                               'jboss_brms': False,
+                               'jboss_eap': False,
+                               'jboss_fuse': True},
+                           'max_concurrency': 25},
+                       'scan_type': 'inspect',
+                       'sources': [{'id': source_output['id'],
+                                    'name': source_name,
+                                    'source_type': 'network'}]}
+
+    assert expected_result == scan_show_result
+
+
+@pytest.mark.skip
+def test_edit_scan(isolated_filesystem, qpc_server_config, source):
+    """Edit a single source scan.
+
+    :id: 5ad22515-2276-48b5-a896-f26f039134fa
+    :description: Create a single source scan with default options
+        then edit to use non-default options.
+    :steps:
+        1) Run ``qpc scan add --sources <source>``
+        2) Run ``qpc scan edit --name <name> --disable-optional-products <optional-product>`` # noqa
+    :expectedresults: The edited scan matches specified options for options.
+    :caseautomation: notautomated
+    """
+    pass
 
 
-def test_scan_with_disabled_products(isolated_filesystem, qpc_server_config):
+@pytest.mark.skip
+def test_edit_scan_with_options(isolated_filesystem,
+                                qpc_server_config, source):
     """Perform a scan and disable an optional product.
 
-    :id: b1cd9901-44eb-4e71-846c-34e1b19751d0
+    :id: 29e36e96-3682-11e8-b467-0ed5f89f718b
     :description: Perform a scan and disable an optional product.
-    :steps: Run ``qpc scan start --sources <source> --disable-optional-products
+    :steps:
+        1) Run ``qpc scan start --sources <source> --disable-optional-products
```

Comment:
I thought this module would not have any running scans?

Reply:
Copy/paste error.  Fixed it now.